src/components/video/index.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from "react";
+import PropTypes from "prop-types";
+import EmbedContainer from "react-oembed-container";
+
+const COMPONENT_CLASS_NAME = "c-video";
+
+// simple util function
+function convertStringToNode(string) {
+	const parser = new DOMParser();
+	const doc = parser.parseFromString(string, "text/html");
+	// get the body, will return <body> around your code
+	return doc.body;
+}
+
+// taking in embed code
+// adding on any powa fields
+// todo: return the embed code with the correct statuses
+const getEmbedHTMLWithPlayStatus = (embedMarkup) => {
+	if (embedMarkup) {
+		const embedHTMLWithPlayStatus = convertStringToNode(embedMarkup);
+
+		// innerhtml ensures body tag not rendered
+		return embedHTMLWithPlayStatus.innerHTML;
+	}
+	return "";
+};
+
+const Video = ({ className, aspectRatio, viewportPercentage, embedMarkup }) => {
+	useEffect(() => {
```

Comment:
should I be worried about calling powaBoot excessively here? 

@mapoulos We're migrating our video player and want to make sure we don't have unnecessary or unperformant code here https://github.com/WPMedia/engine-theme-sdk/blob/7a9b7db1b3bce3559bcb14f0728683a7299188fe/src/components/VideoPlayer/index.tsx